cli.js

```diff
@@ -10,17 +10,19 @@ Usage:
 	casket [--name my-little-server] [--dir ~/path/to/dir]
 
 Options:
-    --name     -n  The name of the server, as shown in the GUI.
-    --dir      -d  The directory to serve, default is the current directory.
-    --readonly -r  Do not allow deletion of files & uploads.
-    --port     -p  Default is 8000.
+    --name      -n  The name of the server, as shown in the GUI.
+    --dir       -d  The directory to serve, default is the current directory.
+    --readonly  -r  Do not allow deletion of files & uploads.
+    --no-delete -w  Do not allow deletion of files.
+    --no-upload -u  Do not allow file upload.
+    --port      -p  Default is 8000.
 
 `
 
 
 
 const argv = mri(process.argv.slice(2), {
-	boolean: ['help', 'h', 'readonly', 'r']
+	boolean: ['help', 'h', 'readonly', 'r', 'no-delete', 'w', 'no-upload', 'u']
```

Comment:
Not sure if this is necessary anymore. Otherwise every LGTM!

Reply:
I don't think it is because of the `=== false`, but I think it makes it more consistent with the other boolean options